### ISSUE TITLE
Allow ability to set annotations in the daemonSet in Helm Chart

### ISSUE THREAD

**Issue by derrickburns** [SumoLogic/fluentd-kubernetes-sumologic#139]
To use the [reloader](https://github.com/stakater/Reloader), I need to add an annotation to the daemon set.  For fully automated deployment, I need to do this via the helm chart.

Please add the ability to provide annotations on daemon sets.

**Comment by Constantin07** [comment]
@derrickburns looks like this is already supported https://github.com/helm/charts/blob/master/stable/sumologic-fluentd/values.yaml#L8

**Comment by derrickburns** [comment]
No, to be clear, I am asking for the ability to set annotations on the daemonset resource itself.

**Comment by Constantin07** [comment]
@derrickburns Thanks for clarification. I've created a PR here to address your request https://github.com/helm/charts/pull/17055
